kalite/distributed/tests/browser_tests/coachreports.py

```diff
@@ -23,71 +21,100 @@ def setUp(self):
 
         self.facility = self.create_facility()
 
+        self.group = FacilityGroup(name="Test Group", facility=self.facility)
+        self.group.save()
+
+        self.url = self.reverse("tabular_view")
+        self.topic = "addition-subtraction"
+
     def test_no_groups_no_topic_selected(self):
         self.browser_login_admin(**self.admin_data)
-        self.browse_to(self.reverse("tabular_view") + "?topic=addition-subtraction")
-        self.browser.find_element_by_css_selector('#error_message')
-        self.assertEqual(self.browser.find_element_by_css_selector('#error_message').text, _("No learner accounts have been created."), "Error message with no users available, no topic selected.")
+        url = "%s?topic=%s" % (self.url, self.topic,)
+        self.browse_to(url)
+        try:
+            elem = self.browser.find_element_by_css_selector('.alert-danger')
```

Comment:
I think it's best to use a `WebDriverWait` here, just to hedge our bets against a race condition. For example, this line might throw a `NoSuchElementException` if the page hasn't finished loading. You can replace it with
```python
elem = WebDriverWait(self.browser, 3).until( EC.presence_of_element_located((By.CLASS_NAME, "alert-danger"))
```

And then change the `except NoSuchElementException` to `except TimeoutException` on line 38, since that's what will be thrown if the element is not found.